hoptimator-kafka/src/test/resources/kafka-ddl.id

```diff
@@ -24,8 +24,8 @@ spec:
   job:
     entryClass: com.linkedin.hoptimator.flink.runner.FlinkRunner
     args:
-    - CREATE TABLE IF NOT EXISTS `existing-topic-2` (`KEY` VARCHAR, `VALUE` BINARY) WITH ('properties.bootstrap.servers'='localhost:9092', 'topic'='existing-topic-2', 'connector'='kafka')
-    - CREATE TABLE IF NOT EXISTS `existing-topic-1` (`KEY` VARCHAR, `VALUE` BINARY) WITH ('properties.bootstrap.servers'='localhost:9092', 'topic'='existing-topic-1', 'connector'='kafka')
+    - CREATE TABLE IF NOT EXISTS `existing-topic-2` (`KEY` VARCHAR, `VALUE` BINARY) WITH ('properties.bootstrap.servers'='localhost:9092', 'connector'='kafka', 'topic'='existing-topic-2')
+    - CREATE TABLE IF NOT EXISTS `existing-topic-1` (`KEY` VARCHAR, `VALUE` BINARY) WITH ('properties.bootstrap.servers'='localhost:9092', 'connector'='kafka', 'topic'='existing-topic-1')
```

Comment:
Not sure why this order changed, was causing tests to fail locally

Reply:
I recently added `.sorted()` to `QuidemTestBase` so this shouldn't happen going forward. These were non-deterministic previously.

Reply:
I reverted this and it's passing now anyway